src/hooks/useFocusWithin/useFocusWithin.ts

```diff
@@ -96,6 +96,17 @@ export function useFocusWithin(props: UseFocusWithinProps) {
                 return;
             }
 
+            if (
+                document.activeElement === event.target &&
+                (event.target instanceof HTMLButtonElement ||
+                    event.target instanceof HTMLInputElement ||
+                    event.target instanceof HTMLTextAreaElement ||
+                    event.target instanceof HTMLSelectElement) &&
+                !event.target.disabled
+            ) {
+                return;
+            }
+
```

Comment:
Let's remove this and add `document.activeElement !== event.target && (...)` to that check https://github.com/gravity-ui/uikit/blob/4c3d070a74cee72a5017ac6df37b3f8cf22b2122/src/hooks/useFocusWithin/useFocusWithin.ts#L187-L191

Reply:
Done.